lib/view/maalcie/beheer/BeheerMaaltijdenBeoordelingenLijst.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace CsrDelft\view\maalcie\beheer;
+
+use CsrDelft\model\entity\maalcie\Maaltijd;
+use CsrDelft\model\maalcie\MaaltijdBeoordelingenModel;
+use CsrDelft\model\ProfielModel;
+use CsrDelft\view\datatable\DataTableResponse;
+
+class BeheerMaaltijdenBeoordelingenLijst extends DataTableResponse {
+	/**
+	 * @param Maaltijd $maaltijd
+	 *
+	 * @return string
+	 */
+	public function getJson($maaltijd) {
+		$data = $maaltijd->jsonSerialize();
+
+		// Haal beoordelingsamenvatting op
+		$stat = MaaltijdBeoordelingenModel::instance()->getBeoordelingSamenvatting($maaltijd);
+		$data['aantal_beoordelingen'] = $stat->kwantiteit_aantal . ", " . $stat->kwaliteit_aantal;
+		$data['kwantiteit'] = $this->getalWeergave($stat->kwantiteit, 'n.a.', 3);
+		$data['kwaliteit'] = $this->getalWeergave($stat->kwaliteit, 'n.a.', 3);
+		$data['kwantiteit_afwijking'] = $this->getalWeergave($stat->kwantiteit_afwijking, 'n.a.', 3, true);
+		$data['kwaliteit_afwijking'] = $this->getalWeergave($stat->kwaliteit_afwijking, 'n.a.', 3, true);
+
+		// Haal koks op
+		$kokTaken = $maaltijd->getCorveeTaken('Kwalikok');
```

Comment:
Kleine vraag over deze keuze: ik heb de naam van de functie _Kwalikok_ hard in de code gezet. Ik kon geen manier vinden hoe ik deze dynamisch kon achterhalen.
Aangezien deze functie niet snel zal veranderen en dit scherm alleen voor MaalCie gebruikt wordt, leek me dit de beste oplossing. Een van jullie nog andere ideeën?

Reply:
Ik heb de naam omgezet naar een ID en deze als constante gedefinieerd in CorveeFunctie-entity.

Reply:
Ik heb het ID van de kwalikok op productie niet, dus deze moet voor uitrol nog even gedefinieerd worden.

Reply:
@jptned de id voor kwalikok is `7`

Reply:
@qurben Aangepast ✔